From f834596de301c0e671773337adc41462370262d7 Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Tue, 11 Aug 2026 15:03:08 +0000
Subject: [PATCH] Keyguard: Default to small clock (1/3)

It looks alright actually, but as always breaks under landscape

[Port to A16] Unchanged one-line default; restored so the retained
ThemePicker (3/3, manifest 276) and DeskClock (2/3, manifest 277) halves agree
with the framework default.

Change-Id: I390c08c043fe516da9b7ed77d60093f2269fb939
---
 core/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index 892bcadbc099..c6086e62c2a0 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -1518,7 +1518,7 @@
     <!-- Provides default value for double line clock in lock screen setting:
          0 - Don't show double line clock
          1 - Show double line clock in lock screen (default) -->
-    <integer name="config_doublelineClockDefault">1</integer>
+    <integer name="config_doublelineClockDefault">0</integer>
 
     <!-- If true, enables verification of the lockscreen credential in the factory reset protection
         flow. This should be true if gatekeeper / weaver credentials can still be checked after a
-- 
2.43.0

